docs/source/release/v3.12.0/framework.rst

```diff
@@ -68,6 +68,11 @@ Core Functionality
 - Fixed the behaviour of ``UpdateInstrumentDefinitions.OnStartup`` in the :ref:`properties file <Properties File>`. It was not being used correctly for using the updated ``Facilities.xml`` file.
 - ``MultiFileProperty`` now accepts complex summation ranges for run numbers, such as ``111-113+115`` and ``111-115+123-132``.
 
+Live Data
+---------
+
+- `KafkaEventListener` is a new live listener for neutron event and sample environment data which is in development for the ESS and ISIS.
```

Comment:
How does this entry reflect the changes in this PR?

Anyway, I think you need double `` around the KafkaEventListener word, otherwise those little guys don't have any effect. See for example the the other entries in this file.

Reply:
As I mentioned in the PR description, the addition to the release notes is because the changes make the listener visible to users for the first time. Previously it was only accessible by selecting `TEST_LIVE` as the default facility, as it was not deployed on real instruments.

Single backticks makes the text italics. I agree double backticks should be used in this case, I added them in a couple of other places too.

Reply:
I see, should've read the description more carefully. Sorry for the noise!

The single backticks don't seem to have any effect when I build the html docs, and I didn't find any reference to them in sphinx docs. If you want italics (emphasis), you should use `*`, they say.

Reply:
Strange, looks like this for me:
![screenshot from 2018-03-01 09-13-45](https://user-images.githubusercontent.com/13455433/36836351-fae34a68-1d30-11e8-8bd7-1cc69ceaa26f.png)

Reply:
Strange, indeed.